### ISSUE TITLE
Wrong "Seen" number while showing pokedex details

### ISSUE THREAD

**Issue by SadisticMystic** [BernardoGiordano/PKSM#419]
4.3.0 brings the new feature to see your current game's Pokedex completion level from within the bank, and it seems to get everything right, except when it comes to tabulating the whole Pokedex: if it says you have 380 Seen and 513 Caught, something's clearly off. Counting the silhouettes, it's clear they should simply be the other way around, which is at least an easy fix.

**Comment by BernardoGiordano** [comment]
thing is that labels aren't switched. The functions come from PKHeX, so I don't know what's wrong

**Comment by clavin** [comment]
I just tested this on my Moon save and it gave the right numbers (that is, *seen* was higher than *caught*). The same goes for my Omega Ruby and X saves.

So I also checked the i18n files to see if they're all in the correct order (with some Google Translate help), and they indeed are all in the order *caught* on line 558 followed by *seen* on line 559.

So, for all practical purposes, this issue is **not reproducible** without a save file from the issue creator.

The only possible way to reproduce this currently is to use a save in which multiple pokemon have their *caught* flag set, but none of their *seen* flags set (which doesn't happen naturally in-game). *In fact,* using PKHeX I set some *caught* flags without setting *seen* flags and found that I was able to artificially inflate my *caught* count without increasing my *seen* count (although I already expected this: [the counts are separate anyway](https://github.com/BernardoGiordano/PKSM/blob/6b07f0f529093c2ac701bea9f7c1b829a53ce65d/source/bank.c#L419-L420)).

---

Also, while checking the i18n files, my Google Translate & Google search skills tells me that "seen" in Italian is `visto`, rather than the [current `visti`](https://github.com/BernardoGiordano/PKSM/blob/bb9b5dd4575c5817b0ff00af81bedcc6be4b84f3/assets/romfs/i18n/it/app.txt#L559)--but, unlike *some* maintainers, I don't know Italian, and could be wrong. ¯\\\_(ツ)_/¯

**Comment by GM-Script-Writer-62850** [comment]
TLDR: There is definably a bug with seen and it is not just the total


I am able to reproduce this on my moon game, but not my sun game (national dex related?)
In my case i have 264 seen and 368 caught, i did completed the 300 in game dex
By comparing my national dex in pkhex and pksv 4.4
*I will be assuming grayed out is seen full color is caught and black silhouette is not seen/caught
There is a bug somewhere
- PKsv: Bulbasaur (Caught)
- PKhex: Bulbasaur (Caught/Seen)
- PKsv: Ivysaur (Seen)
- PKhex: Ivysaur (NOT seen or caught)
- PKsv:Venasaur (Seen)
- PKhex: Venasaur (Seen)

- PKsv: Squirtle (NOT seen)
- PKhex: Squirtle (NOT seen)
- PKsv: Warturtle (Seen)
- PKhex: Warturtle (NOT seen)
- PKsv: Blastoise (Seen)
- PKhex: Blastoise (Seen)

- PKsv: Weedle (NOT seen)
- PKhex: Weedle (seen)
- PKsv: Kakuna (Seen)
- PKhex: Kakuna (NOT seen or caught)
- PKsv: Beedrill (Not Seen)
- PKhex: Beedrill (NOT seen or caught)

- PKsv: Ekans (NOT seen)
- PKhex: Ekans (seen)
- PKsv: Arbok (NOT seen)
- PKhex:Arbok (NOT seen or caught)

- Pkhex Bellsprout (NOT seen or caught)
- Pksv: Bellsprout (Seen)
- Pkhex Weepinbell (NOT seen or caught)
- Pksv: Weepingbell (Not seen)
- Pkhex Victorybell (NOT seen or caught)
- Pksv: Victorybell  (Seen)

- Pkhex: Mewtwo (Not seen)
- PKsv: Mewtwo (Seen)

- Pkhex: Mew (Not seen)
- PKsv: Mew (Not Seen)

- Pkhex Ralts (seen/caught)
- Pksv: Ralts (Caught)
- Pkhex Kirlia (NOT seen or caught)
- Pksv: Kirlia (Seen)
- Pkhex Guadavur (NOT seen or caught)
- Pksv: Guadavur  (Seen)

Now that we have seen that lets take a closer look at my sun Save as it does not have a complete aloha dex
In game pokdex:
- Bagon (Seen)
- Shelgon/Slamance (Not seen)
PKSV:
- Bagon/Shelgon (Not seen)
 - Salamance (Seen)

In game pokdex:
- Pikipek/Trumbeak (Seen)
- Toucannon (Not seen)
PKSV: 
- Pikipek (Not seen)
- Trumbeak/Toucannon (Seen)
